### PR TITLE
feat(greengrasssdk.py): add greengrass support

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: epsagon/events/greengrasssdk.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+from __future__ import absolute_import
+import traceback
+from uuid import uuid4
+
+from epsagon.utils import add_data_if_needed
+from ..trace import trace_factory
+from ..event import BaseEvent
+
+
+class GreengrassEvent(BaseEvent):
```
Comment (maorlx) [inline epsagon/events/greengrasssdk.py:14]: consider renaming to greengrass publish event

File: epsagon/events/greengrasssdk.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
+"""
```
Comment (maorlx) [inline epsagon/events/greengrasssdk.py:1]: looks good : ) 
can you add a sanity UT test?

Comment (ranrib) [reply epsagon/events/greengrasssdk.py:1]: 👍

File: epsagon/events/greengrasssdk.py (hunk 1 above)
Comment (ranrib) [reply epsagon/events/greengrasssdk.py]: 👍

**Review by maorlx — APPROVED** [review]

**Pull request merged**